python/tvm/contrib/cc.py

```diff
@@ -207,66 +204,31 @@ def _linux_compile(output, objects, options, compile_cmd="g++", compile_shared=F
         msg += "\nCommand line: " + " ".join(cmd)
         raise RuntimeError(msg)
 
-
 def _windows_shared(output, objects, options):
```

Comment:
This re-write with clang is a lot cleaner, and aligns with https://github.com/apache/incubator-tvm/blob/master/apps/cpp_rpc/rpc_env.cc#L232-L246 which was tested successfully on windows.